### ISSUE TITLE
Wrong include dir in pkg-config persists

### ISSUE THREAD

**Issue by ctreffs** [assimp/assimp#3174]
Hi,

sadly with assimp 5.0.1 the problem still persists and moreover the pkg-config file provides the wrong version in the description.

```
prefix=/usr/local/Cellar/assimp/5.0.1
exec_prefix=/usr/local/Cellar/assimp/5.0.1/
libdir=/usr/local/Cellar/assimp/5.0.1/lib
includedir=/usr/local/Cellar/assimp/5.0.1/../include/include

Name: Assimp
Description: Import various well-known 3D model formats in an uniform manner.
Version: 5.0.0
Libs: -L${libdir} -lassimp
Libs.private:  -lz
Cflags: -I${includedir}

```
The `includedir` property should look like this:
 `includedir=/usr/local/Cellar/assimp/5.0.1/include`
and the package version should obviously be 5.0.1 rather than 5.0.0.

_Originally posted by @ctreffs in https://github.com/assimp/assimp/issues/2804#issuecomment-620430488_

**Comment by RichardTea** [comment]
These have all been fixed since the 5.0.1 release tag, please check current master.

The include path had an interim fix in Feb 2020, and had a better fix about three weeks ago https://github.com/assimp/assimp/commit/9fb81c3be6dc30139ea32c92046c9794dca0e73e#diff-c22480c55c406da6bdf8668a4a4fca1a

The version number was also corrected a few weeks ago.

**Comment by kimkulling** [comment]
I will release a new bugfix release 5.0.2 to deploy this.
